gtk3/sample/misc/tooltips.rb

```diff
@@ -5,28 +5,27 @@
 
   Copyright (c) 2007-2015 Ruby-GNOME2 Project Team
   This program is licenced under the same licence as Ruby-GNOME2.
-
-  $Id: tooltips.rb,v 1.1 2007/07/10 13:17:34 ggc Exp $
 =end
 
-require 'gtk3'
+require "gtk3"
 
-unless Gtk::Version.or_later?(2, 12, 0)
-    puts "This sample requires GTK+ 2.12.0 or later"
-    exit
+unless Gtk::Version.or_later?(3, 4, 2)
+  puts "This sample requires GTK+ 3.4.2 or later: #{Gtk::Version::STRING}"
+  exit
 end
 
+
 def treeview_query_tooltip(treeview, keyboard_tip, x, y, tooltip)
     if keyboard_tip
         # Keyboard mode
-        path, = treeview.cursor
+        ok,path = treeview.cursor
```

Comment:
For this one you are right, I was misled by the other error  below which stop the script when I tryed to update it